indico/modules/events/surveys/models/submissions.py

```diff
@@ -131,3 +135,16 @@ class SurveyAnswer(db.Model):
     @return_ascii
     def __repr__(self):
         return '<SurveyAnswer({}, {}): {}>'.format(self.submission_id, self.question_id, self.data)
+
+    @property
+    def answer_data(self):
+        field_type = self.question.field_type
+        if field_type in {'multiselect', 'single_choice'}:
```

Comment:
can we do this in a cleaner way without referencing specific field types here? maybe a method on the field implementation?